.github/workflows/check-catalyst.yaml

```diff
@@ -42,9 +42,15 @@ jobs:
     steps:
       - name: Checkout Catalyst repo
         uses: actions/checkout@v4
+      
+      - name: Set up Python # Ensure python3.10 is used
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ needs.constants.outputs.primary_python_version }}
 
       - name: Install dependencies
         run: |
+          python3 --version
```

Comment:
Is this just for informational purposes?
I think the setup-python step above already reports that information.